src/components/terms/legal-list.css

```diff
@@ -0,0 +1,19 @@
+/* From https://davelane.nz/creating-legal-list-numbering-only-using-css */
```

Comment:
let's use css modules => https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/

Reply:
I have deleted the css file as it was uncessary

Reply:
reinstated css file